lard_tests/src/bin/prepare_db.rs

```diff
@@ -0,0 +1,328 @@
+// use std::env;
+use std::fs;
+
+use chrono::{DateTime, Duration, DurationRound, TimeDelta, TimeZone, Utc};
+use tokio_postgres::{Client, Error, NoTls};
+
+const CONNECT_STRING: &str = "host=localhost user=postgres dbname=postgres password=postgres";
+
+struct Param<'a> {
+    id: i32,
+    code: &'a str,
+}
+
+// TODO: maybe merge into fake_data_generator, a lot of the code is shared
+struct Labels<'a> {
+    // Assigned automatically
+    // timeseries: i32,
+    station_id: i32,
+    param: Param<'a>,
+    type_id: i32,
+    level: Option<i32>,
+    sensor: Option<i32>,
+}
+
+struct Location {
+    lat: f32,
+    lon: f32,
+    // hamsl: f32,
+    // hag: f32,
+}
+
+struct Timeseries {
+    from: DateTime<Utc>,
+    period: Duration,
+    len: i32,
+    deactivated: bool,
+    loc: Location,
+}
+
+impl Timeseries {
+    fn end_time(&self) -> DateTime<Utc> {
+        self.from + self.period * self.len
+    }
+}
+
+struct Case<'a> {
+    title: &'a str,
+    ts: Timeseries,
+    meta: Labels<'a>,
+}
+
+async fn create_single_ts(client: &Client, ts: Timeseries) -> Result<i32, Error> {
+    let end_time = ts.end_time();
+
+    // Insert timeseries
+    let id = match ts.deactivated {
```

Comment:
help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#match_bool
help: consider using an `if`/`else` expression